gdp/column/column.py

```diff
@@ -113,7 +113,7 @@ def tray_no_tray(b, t):
             doc='Vapor fraction of feed',
             initialize=init['feed_vap_frac'], bounds=(0, 1))
     
-        # Component feed flow variable (mol/s)
+        # Component feed flow variable [mol/s]
```

Comment:
It is still unclear whether the flows are all [mol/s] or [kmol/s]. Please clarify.

Reply:
According to the Paper from (Ghouse et. al., 2018) He specified the feed to be as 100 mol/s. Moreover, every flow notations beside the feed were [kmol] when I observed the dsda-gdp.

Reply:
Changed into all into [mol/s] for unified the flow.